src/Data/Text/Builder/Linear/Buffer.hs

```diff
@@ -133,13 +133,13 @@ buffer |># addr# =
 -- >>> runBuffer (\b -> "foo"# #<| "bar"# #<| b)
 -- "foobar"
 --
--- The literal string must not contain zero bytes @\\0@ and must be a valid UTF-8,
+-- The literal string must not contain zero bytes @\\NUL@ and must be a valid UTF-8,
 -- these conditions are not checked.
 --
 -- /Note:/ When the syntactic extensions @UnboxedTuples@ or @UnboxedSums@ are
 -- enabled, extra spaces are required when using parentheses: i.e. use @( '#<|' )@
 -- instead of @('#<|')@. See the GHC User Guide chapter
--- “[Unboxed types and primitive operations](https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/primitives.html#unboxed-tuples)”
+-- “<https://downloads.haskell.org/ghc/latest/docs/users_guide/exts/primitives.html#unboxed-tuples Unboxed types and primitive operations>”
```

Comment:
Any particular reason for this change?

Reply:
For some reason it [is not parsed by haddock](https://hackage.haskell.org/package/text-builder-linear-0.1.2/docs/Data-Text-Builder-Linear-Buffer.html#v:-35--60--124-)